pipelines/dotnet.yml

```diff
@@ -41,35 +135,44 @@ steps:
     packagesToPack: '$(package)'
     packDestination: '$(Build.ArtifactStagingDirectory)'
     includeSymbols: true
-  
-- task: EsrpCodeSigning@4
-  displayName: 'Sign'
+    nobuild: true
+    configuration: '$(buildConfiguration)'
+
+- task: EsrpCodeSigning@5
+  displayName: 'Sign Nuget package'
   inputs:
-    ConnectedServiceName: 'Teams ESRP CodeSign v3'
+    ConnectedServiceName: 'TeamsESRP-CP-401405'
+    AppRegistrationTenantId: cdc5aeea-15c5-4db6-b079-fcadd2505dc2
+    AppRegistrationClientId: a94a770a-9a7b-4888-a3ea-24584b851e49
```

Comment:
double checking- are these all okay as public vals?

Reply:
https://github.com/microsoft/OpenAPI.NET/blob/25136afa5a35160c75ab6e8bd297fbab479efd74/.azure-pipelines/ci-build.yml#L85

I found a handful of OSS repos that have these ids publically available. I think it should be fine.